src/use/helpers/Toasts.ts

```diff
@@ -1,9 +1,21 @@
 //import ReportBug from "@/components/common/ReportBug.vue";
 //import { markRaw } from "vue";
+//import Router from "@/use/router";
 import { useToast } from "@/toast";
 
 const toast = useToast();
 
+// function routeToDashboard() {
+//     Router.push({ name: "welcome" });
+// }
+
```

Comment:
remove comments from this file

Reply:
I removed all commented-out lines in this file (they caused cypress errorer regarding our webpack conf) https://github.com/upb-uc4/ui-web/pull/785/commits/31e64ed58ab1ab0fa38bc081c366dc629a0350ef